update_compilers/install_compilers.sh

```diff
@@ -461,7 +461,16 @@ get_ispc 1.9.2
 get_ispc 1.9.1
 
 # PPCI
-pip3 install ppci
+get_ppci() {
+  local VER=0.5.4
```

Comment:
Could this not be hardcoded so we could call it like `get_ppci X.Y.Z` in case we ever want to add another version?

Reply:
Yes, I finally figured out how to do this :)